src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -339,14 +348,8 @@ $externalMargin: 8px;
 	height: var(--header-height);
 
 	&__trigger {
-		display: flex;
-		align-items: center;
-		justify-content: center;
 		width: var(--header-height);
 		height: var(--header-height);
-		margin: 0;
-		padding: 0;
-		cursor: pointer;
 		opacity: .85;
 		background-color: transparent;
```

Comment:
I think this is no longer needed due to `tertiary-no-background`

Reply:
Gotcha! Will remove right now :)

Reply:
Sorry for the delay - has been removed now :D